frontend/ui/data/onetime_migration.lua

```diff
@@ -341,7 +341,7 @@ if last_migration_date < 20220116 then
         if not found then
             local std_ebooks = {
                 title = "Standard Ebooks",
-                url = "https://standardebooks.org/opds",
+                url = "https://standardebooks.org/feeds/opds",
```

Comment:
@NiLuJe Can you sanity check? I kept this change since it seems a bit silly otherwise, or do you think the migrations should remain "pure" so that the new one down below will do it.

Reply:
I don't have strong feelings about either approaches. I'd probably have left this one alone myself, but it's fairly harmless in the grand scheme of things ;).

Reply:
I would've never changed this line. I'll change it back too then, lol.